packages/web-components/src/components/ic-popover-menu/ic-popover-menu.css

```diff
@@ -7,8 +7,8 @@
 
 @media (prefers-reduced-motion: no-preference) {
   :host([open]) {
-    opacity: 1;
-    transition: opacity var(--ic-transition-duration-fast);
+    display: block;
+    transition: display var(--ic-transition-duration-fast);
```

Comment:
Is there any animation on `display block`? If the transition property is removed, what happens?